### DISCUSSION TITLE
Create a new folder "Bit Manipulation" in DSA

### DISCUSSION THREAD

**Discussion by tanyagupta0201** [AkashSingh3031/The-Complete-FAANG-Preparation#175] [General]
Can I create a new folder Bit Manipulation in Data Structures section ?? I have some problems to add there.

**Comment by AkashSingh3031** [comment]
> Can I create a new folder Bit Manipulation in Data Structures section ?? I have some problems to add there.

Ok, you can create but mention number also in sequence

**Reply by tanyagupta0201** [reply]
Yeah sure
